search-extension/src/main/java/com/afrigis/services/search/extension/CensusGetType.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.afrigis.services.search.extension;
+
+/**
+ *
```

Comment:
@maposatj please just flesh out your javadocs a bit - briefly explain what this enum is for?